lib/rust/shapely/macros/src/derive_entry_point.rs

```diff
@@ -7,8 +7,6 @@ use crate::prelude::*;
 // ===================
 
 fn crate_name_to_base_name(name: &str) -> String {
-    let name = name.replace("debug-scene-", "");
-    let name = name.replace("ensogl-example-", "");
```

Comment:
We no longer have special entry points, so I think this entire module could be removed.